openhantek/src/post/graphgenerator.cpp

```diff
@@ -53,14 +53,14 @@ void GraphGenerator::generateGraphsTYvoltage(PPresult *result) {
         // Set size directly to avoid reallocations
         target.resize(sampleCount);
 
-        // Data samples are in [0,1] (as long as the voltageLimits are set correctly).
+        // Data samples are in volts (as long as the voltageLimits are set correctly).
```

Comment:
There seems to be a bit of confusion going on about what's stored in these vectors.
Most of the code seems to assume that it is actually a voltage in volts.